package.json

```diff
@@ -121,6 +121,6 @@
     "reselect": "^4.0.0",
     "sass": "^1.56.2",
     "unchained-bitcoin": "^0.3.0",
-    "unchained-wallets": "^0.2.3"
+    "unchained-wallets": "bucko13/unchained-wallets#ledger-address-verification"
```

Comment:
Just trying to wrap my head around the order of operations:
- we merge that unchained-wallets PR first
- we change this line to 0.2.4 (or whatever the new release version is)
?

Reply:
This has been published and updated